src/Roslyn.Suppressions/Roslyn.Suppressions/SonarDiagnosticSuppressor.cs

```diff
@@ -70,7 +73,7 @@ public override void ReportSuppressions(SuppressionAnalysisContext context)
             {
                 return Enumerable.Empty<Suppression>();
             }
-
+            CodeMarkers.Instance.GetSuppressionsStart();
```

Comment:
Currently, this is going report as follows:
In standalone mode:
```
ReportSuppressionStart
ReportSuppressionStop
```

i.e. we have to infer that we are in standalone mode by the absence of the other events.

In connected mode:
```
ReportSuppressionStart
GetSuppressionsStart
GetSuppressionsStop(n)
ReportSuppressionStop
```
The two extra events are adding much from a timing point of view since they are called very close to the `ReportSuppressionStart/Stop`.

It would be simpler and more useful if we always reported something like:
```
ReportSuppressionStart
ReportSuppressionStop (mode, suppressionCount)
```